contracts/nois-proxy/src/msg.rs

```diff
@@ -67,6 +67,22 @@ impl From<ProxyExecuteMsg> for ExecuteMsg {
     }
 }
 
+#[cw_serde]
+pub enum SudoMsg {
+    /// Withdraw the given amount to the withdrawal address
+    #[cfg(feature = "governance_owned")]
+    Withdaw { amount: Coin, address: String },
+    /// Withdraw all available balance of this token to the withdrawal address
+    #[cfg(feature = "governance_owned")]
+    WithdawAll { denom: String, address: String },
+    /// Withdraw the given amount to the community pool
+    #[cfg(feature = "governance_owned")]
+    WithdrawToCommunityPool { amount: Coin },
+    /// Withdraw all available balance of this token to the community pool
+    #[cfg(feature = "governance_owned")]
+    WithdrawAllToCommunityPool { denom: String },
+}
```

Comment:
This will cover pretty much every use case, I'd like to see the `SetConfig` as sudo message as well so governance can update the price

Reply:
Doing it right now